invenio_cli/commands/upgrades.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+from os import listdir
+
+from ..helpers.process import ProcessResponse, run_cmd, run_interactive
+from .steps import CommandStep
+
+
+class UpgradesCommands(object):
```

Comment:
what about
```suggestion
class UpgradeCommands(object):
```

Same for the file name (`upgrade.py`)